src/plugins/controls/public/control_group/control_group_strings.ts

```diff
@@ -56,6 +64,10 @@ export const ControlGroupStrings = {
       i18n.translate('controls.controlGroup.manageControl.widthInputTitle', {
         defaultMessage: 'Control size',
       }),
+    getControlSettingsTitle: () =>
+      i18n.translate('controls.controlGroup.manageControl.controlSettingsTitle', {
+        defaultMessage: 'Control settings',
```

Comment:
I wonder if we should use the friendly name of the selected Control Type here, so it would read `Options List settings` or `Range slider settings`

Reply:
Great point! @andreadelrio what are your thoughts?

Reply:
I'm thinking if we need to show that label at all? `Label` and `Width` are settings too right so no need to have a separate section for (other) settings? or are you trying to keep the toggle for width under `Minimum width` and so you need a heading for the rest of the settings?
<img width="309" alt="image" src="https://user-images.githubusercontent.com/4016496/168931787-8fbce0a9-4e01-4504-8b55-dde660b05e72.png">

Reply:
@andreadelrio Yeah, it seemed to me like grouping the type-specific control settings with the `Minimum width` settings was confusing - so either the type-specific settings should go **above** the `Minimum width` (which would cause shifting when switching between `Options list` and `Range slider` fields) or I think these type-specific settings should have their own label.

What do you think, though?

Reply:
I think we could either group them under something like **Additional settings**? or give each toggle its own title so something like:

- **Multiple selections**

Allow multiple selections in dropdown

- **Timeout**

Run past timeout

What do you think @KOTungseth ?

Reply:
I like `Additional settings` but definitely interested in hearing @KOTungseth's thoughts! :)